generic/tests/cfg/unattended_install_reboot_driftfix.cfg

```diff
@@ -0,0 +1,58 @@
+- unattended_install_reboot_driftfix:
+    virt_test_type = qemu libvirt
+    type = unattended_install_driftfix
+    no WinXP Win2000 Win2003 WinVista
+    only Windows
+    extra_params += "-global kvm-pit.lost_tick_policy=delay"
+    backup_image_after_testing_passed = yes
+    start_vm = no
+    kill_vm = yes
+    kill_vm_gracefully = yes
+    kill_vm_on_error = yes
+    shutdown_cleanly = yes
+    shutdown_cleanly_timeout = 120
```

Comment:
If no special requirement with the default value of the framework, suggest remove it. Here it is same with the default value.